autonomy/constants.py

```diff
@@ -22,12 +22,20 @@
 
 DEFAULT_BUILD_FOLDER = "abci_build"
 DEFAULT_KEYS_FILE = "keys.json"
-DEFAULT_IMAGE_VERSION = "0.1.0"
-IMAGE_VERSION = os.environ.get("VERSION", DEFAULT_IMAGE_VERSION)
+DEFAULT_IMAGE_VERSION = "latest"
```

Comment:
I think it would be good to allow for explicit version too. Happy for the default being `latest`, but an explicit override should be possible?

Reply:
it's possible using env vars, just below